synthea-hiv/generator/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile used to create gobbledegook/synthea
+
+FROM openjdk:16-jdk-alpine
+RUN apk --no-cache add openssl wget git curl
+
+WORKDIR /synthea-hiv
+RUN wget https://github.com/synthetichealth/synthea/releases/download/master-branch-latest/synthea-with-dependencies.jar
+COPY hiv_simple.json hiv_simple.json
+COPY hiv_simple hiv_simple
+
+ENV POP 100
+
+CMD java -jar synthea-with-dependencies.jar \
+    -p ${POP} \
+    --exporter.years_of_history 0 \
+    --generate.only_alive_patients true \
+    --generate.max_attempts_to_keep_patient 100 \
+    -d ./ \
+    -m hiv_simple \
```

Comment:
Did you try achieving the same goal without `-m`?

Reply:
without the `m`, it includes all modules that are part of Synthea in the patient history